nerfstudio/data/dataparsers/colmap_dataparser.py

```diff
@@ -251,6 +251,13 @@ def _get_image_indices(self, image_filenames, split):
     def _generate_dataparser_outputs(self, split: str = "train", **kwargs):
         assert self.config.data.exists(), f"Data directory {self.config.data} does not exist."
         colmap_path = self.config.data / self.config.colmap_path
+        if not colmap_path.exists():
+            # Try detecting the colmap path automatically
```

Comment:
Personally, I would choose to let it fail instead of manually searching in the pre-defined paths.

Reply:
Maybe first asking users if they want to do this search? Or I can just remove this part. (since this path error happens a lot to me

Reply:
I may misunderstand something here. But I think if there is an path error happening to users, users should be the ones to make the path right.